lib/ansible/modules/files/lineinfile.py

```diff
@@ -158,26 +158,26 @@
     line: '192.168.1.99 foo.lab.net foo'
     create: yes
 
-# Fully quoted because of the ': ' on the line. See the Gotchas in the YAML docs.
+# Double quoted because of the ': ' on the line. See the Gotchas in the YAML docs.
 - lineinfile:
     path: /etc/sudoers
     state: present
     regexp: '^%wheel\s'
-    line: '%wheel ALL=(ALL) NOPASSWD: ALL'
+    line: "%wheel ALL=(ALL) NOPASSWD: ALL"
 
 # Yaml requires escaping backslashes in double quotes but not in single quotes
 - lineinfile:
     path: /opt/jboss-as/bin/standalone.conf
     regexp: '^(.*)Xms(\\d+)m(.*)$'
-    line: '\1Xms${xms}m\3'
+    line: "\1Xms${xms}m\3"
```

Comment:
This example is supposed to show the difference in single vs double quoting requirements in YAML, but it's incorrect. This like should be singel quoted but the `regexp` should be double quoted.